### PR TITLE
Add tests for DCPower Create Session(s) VIs

### PR DESCRIPTION
**Pull request by jonathanmendez**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Adds a test for DCPower `Create Sessions - 1Sess` and a test for `Create Sessions - NSess`.
- Adds an assertion helper for checking number of sessions.

### Why should this Pull Request be merged?

To serve as a pattern for other instrument "Create Sessions" tests and to share the assertion helper.

### What testing has been done?

This adds tests for functionality previously added.

### CODE REVIEW

File: Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib
Comment (pbirkhol-ni) [inline Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib:1]: I don't know how we want to organize our tests, so I'm open to ideas. For Get Connections, I added a separate folder on disk and then also added a new virtual folder in the project. Within that virtual folder, I have sub virtual folders for each driver type.

Comment (pbirkhol-ni) [inline Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib:1]: Make sure to add the tests to the test list.

**Review by pbirkhol-ni — APPROVED** [review]
This seems fine as a basic set of sanity tests that all drivers will implement. I expect there will be much more thorough tests of the VIMs.

File: Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib
Comment (jonathanmendez) [reply Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib:1]: We had some offline discussion about this and for now we'll aim for "organize tests based on functionality first then driver"

Comment (jonathanmendez) [reply Source/Tests/Tests.Runtime/Measurement Server/MeasurementServerTests.lvlib:1]: Done

**Comment by jonathanmendez** [comment]
> This seems fine as a basic set of sanity tests that all drivers will implement. I expect there will be much more thorough tests of the VIMs.

Yes, I'm separately working on implementing these, but wanted to get the pattern established for driver-specific tests.

**Comment by gpachecoNI** [comment]
Should tests be added to the list of test?

**Comment by gpachecoNI** [comment]
> Should tests be added to the list of test?

nvm, I see you already updated it.

**Comment by jonathanmendez** [comment]
For posterity, the changes to the "Create Sessions" VIs (i.e. the non-test changes) are incidental LV quirks. Somehow these VIs held a stale reference to a subVI that had gotten renamed as part of the implementation PR that added them. I resolved the stale references in this branch incidentally.

**Pull request merged**